scripts/build.sh

```diff
@@ -579,7 +579,7 @@ if [ "$QUICK" != "1" ] && [ "$BUILD_DESKTOP$BUILD_MOBILE" != "" ] && ( [[ $QT_VE
 		# regardless of whether we do a fat build or not, let's do the 'native' build here
 		$QMAKE "INCLUDEPATH=$INSTALL_ROOT/../qtlocation/build/include/QtLocation/6.3.0" QMAKE_APPLE_DEVICE_ARCHS="$(uname -m)" ../googlemaps.pro
 	else
-		$QMAKE "INCLUDEPATH=$INSTALL_ROOT/include" ../googlemaps.pro
+		$QMAKE "INCLUDEPATH=$INSTALL_ROOT/include" "CONFIG+=release" ../googlemaps.pro
```

Comment:
I am wondering if we should be adding the same change on https://github.com/subsurface/subsurface/pull/4224/files#diff-3d64996115d0ac3b74e36f57ce6b00d6b4982614c41b1ed39eab8ef43b2e9a2bR580 - after all this is dependent on the Google maps plugin version, and we are using the same version for both paths.

Reply:
No idea.  I don't have a QT 6 build environment so I have not tested to see if maps works or not.

Reply:
I would add it - being explicit here will hopefully stop others from hitting the same (non-failing) bug when they try to build with Qt6.